src/tests/Interop/Swift/InvalidCallingConv.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Swift;
+using Xunit;
+
+public class InvalidCallingConvTests
+{
+    private const string SwiftLib = "libDummy.dylib";
+
+    [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvSwift) })]
+    [DllImport(SwiftLib, EntryPoint = "$s5Dummy9dummyFuncyyF")]
+    public static extern nint DummyFunc1(SwiftSelf self1, SwiftSelf self2);
+
+    [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvSwift) })]
+    [DllImport(SwiftLib, EntryPoint = "$s5Dummy9dummyFuncyyF")]
+    public unsafe static extern nint DummyFunc2(SwiftError* error1, SwiftError* error2);
+
+    [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvSwift) })]
+    [DllImport(SwiftLib, EntryPoint = "$s5Dummy9dummyFuncyyF")]
+    public unsafe static extern nint DummyFunc3(SwiftSelf self1, SwiftSelf self2, SwiftError* error1, SwiftError* error2);
+
+    [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvSwift) })]
+    [DllImport(SwiftLib, EntryPoint = "$s5Dummy9dummyFuncyyF")]
+    public static extern nint DummyFunc4(SwiftError error1);
+
+    [Fact]
+    public unsafe static int TestEntryPoint()
+    {
+        SwiftSelf self = new SwiftSelf(IntPtr.Zero);
+        SwiftError error = new SwiftError(IntPtr.Zero);
+
+        try {
+            DummyFunc1(self, self);
```

Comment:
It would also be helpful to comment as to why. If this fails, what went wrong? These tests don't address that.

Reply:
Added short message for each failure.